misc/backfill_qc_states.py

```diff
@@ -36,12 +37,14 @@ def get_date(date_string, well_metrics):
         qc_date = datetime.strptime(date_string, "%d/%m/%Y")
     else:
         qc_date = well_metrics.run_complete
+        if qc_date is None:
```

Comment:
Could we do something like `qc_date = well_metrics.run_complete or well_complete or well_start or run_start` ?

Reply:
certainly, see [59343cd](https://github.com/wtsi-npg/npg_langqc/pull/124/commits/59343cd0ea2ca6d487943a7d5aab824eb078d40c)